src/ngrx/10-test-redirect-effects/test-redirect-effects.md

```diff
@@ -31,9 +31,42 @@ The `--watch` switch will rerun your tests whenever a code file changes. You can
 
 ## Description
 
-TODO: remove or use parts of for an intro for the section
+When testing Effects, we will verify side-effects are executed properly depending when an appropriate Action is dispatched. In our case, we are working with Effects that cause navigation. We can take advantage of the [RouterTestingModule](https://angular.io/api/router/testing/RouterTestingModule#usage-notes).
```

Comment:
we will verify side-effects are executed properly depending when an appropriate Action is dispatched

it looks like we could drop `depending` and retain same meaning

Reply:
In our case, we are working with Effects that cause navigation (between routes? not sure how this can be improved, but it looks like it is missing something)

Reply:
hmm, I don't know, but I've removed the word "depending"

Reply:
What about "In our case, we are working with Effects that cause navigation, so we can take advantage of the ..."

Reply:
@m-thompson-code Wade's suggestion looks good to me